hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java

```diff
@@ -141,7 +141,7 @@ public static RpcMetrics create(Server server, Configuration conf) {
   MutableCounterLong rpcAuthorizationSuccesses;
   @Metric("Number of client backoff requests")
   MutableCounterLong rpcClientBackoff;
-  @Metric("Number of Slow RPC calls")
+  @Metric("Number of slow RPC calls")
```

Comment:
It looks LGTM overall, but is this change unnecessary?

Reply:
Hi Sir @slfan1989 thanks help me review.
the change here is mainly to consider the naming consistency with some current metrics description.